seed/php-sdk/imdb/tests/Seed/Core/RawClientTest.php

```diff
@@ -1,17 +1,13 @@
 <?php
 
-namespace Seed\Tests\Core;
```

Comment:
I assume this was built off an earlier branch? We'll want to revert the changes in this file once we actually generate things.

Reply:
PHPstorm suggested i change it as i did, interestingly

that being said, i didn't really mean to check that in, so i'll change it back in this PR! but something to talk about for our final state